pephubclient/files_manager.py

```diff
@@ -1,13 +1,20 @@
+import os
 import pathlib
 from contextlib import suppress
-import os
-from typing import Optional
+
+import pandas
+import yaml
+
 from pephubclient.constants import RegistryPath
+from pephubclient.exceptions import PEPExistsError
 
 
 class FilesManager:
```

Comment:
Does this need to be a class? I see below you're using it in a functional way by defining everything as a `staticmethod`. It might be clearer to just take a functional approach here - but most likely is personal preference! Just a thought.

Reply:
https://stackoverflow.com/questions/42757961/a-class-with-all-static-methods
Yeah you are right.

Reply:
I won't change it now, because I a have issue with mocking functions outside class. I will leave it like it is. It shouldn't make big problem.